ledfx/effects/audio.py

```
@@ -416,6 +417,21 @@ def update_config(self, config):
         self._ledfx.config["audio"] = self._config
 
     def activate(self):
+        # Re-entry guard – must be atomic with _class_lock so concurrent
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)